### DISCUSSION TITLE
Controlling width at which sidebar is hidden.

### DISCUSSION THREAD

**Discussion by AlFontal** [quarto-dev/quarto-cli#3198] [Q&A]
I've lately been playing around with Quarto and I am most likely move all of my reporting from Jupyterbook to it, but there is a small issue that keeps irking me and I don't know if it's possible to solve within the Quarto API.

The thing is: I would like to be able to control when the sidebar is hidden/collapsed as a function of page width.
A collapse sidebar button as discussed in #472, would be great too.

The default behavior now is has the sidebar showing both in `floating` and `docked` mode until the page goes under a specific width, when it just is hidden and the main body takes most of the width. The limit, however, is still too slim for my taste, and I would like to be able to set up a higher threshold so that the text is still legible at narrower page widths. 

Is this something possible to control at this point from the `_quarto.yml` config or maybe modifying the site's CSS might solve it?

**Comment by dragonstyle** [comment]
Currently the breakpoint isn't configurable. In 1.3 we expect the sidebar, body, and margin width to be more easily controllable, which may help address this, though it isn't directly changing the breakpoint...

**Reply by AlFontal** [reply]
That's what I thought. Well, having more control over it should at least partially address the issue, looking forward to it!

**Reply by AlFontal** [reply]
sorry @dragonstyle, but has there been any changes on this with the last releases? Thanks!

**Reply by mcanouil** [reply]
Checkout the pre-release (and release) notes on the website: <https://quarto.org/docs/download/>

For instance, in it you have two release notes highlights related to the layout:
- https://quarto.org/docs/prerelease/1.3/grid.html
- https://quarto.org/docs/prerelease/1.3/website-nav.html

**Reply by AlFontal** [reply]
Thanks @mcanouil, I checked it out and indeed now the widths are manually controllable, though as @dragonstyle mentioned, the breakpoint can't be changed yet.
